Resources/doc/basic-usage.rst

```diff
@@ -0,0 +1,94 @@
+Basic Usage
+===========
+
+This bundle works by configuring a set of filters and then applying those
+filters to images inside a template So, start by creating some sort of filter
+that you need to apply somewhere in your application. For example, suppose
+you want to thumbnail an image to a size of 120x90 pixels:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+
+    liip_imagine:
+        resolvers:
+           default:
+              web_path: ~
+
+        filter_sets:
+            cache: ~
+            my_thumb:
+                quality: 75
+                filters:
+                    thumbnail: { size: [120, 90], mode: outbound }
+
+You've now defined a filter set called `my_thumb` that performs a thumbnail transformation.
+We'll learn more about available transformations later, but for now, this
+new filter can be used immediately in a template:
+
+.. code-block:: jinja
+
+    <img src="{{ '/relative/path/to/image.jpg' | imagine_filter('my_thumb') }}" />
+
+Or if you're using PHP templates:
+
+.. code-block:: php
+    
+    <img src="<?php $this['imagine']->filter('/relative/path/to/image.jpg', 'my_thumb') ?>" />
```

Comment:
these can be put together in a code example that uses tabs for different languages:
```rst
.. configuration-block:

    .. code-block:: html+jinja

        <img src="{{ '/relative/path/to/image.jpg'|imagine_filter('my_thumb') }}" />

    .. code-block:: html+php

        <img src="<?php echo $this['imagine']->filter('/relative/path/to/image.jpg', 'my_thumb') ?>" />
```

btw, `/relative/path/to/image.jpg` doesn't look so relative :wink: